equinix-openapi-fabric/pom.xml

```diff
@@ -5,9 +5,9 @@
     <artifactId>equinix-openapi-fabric</artifactId>
     <packaging>jar</packaging>
     <name>equinix-openapi-fabric</name>
-    <version>0.5.0</version>
+    <version>0.10.0</version>
     <url>https://github.com/openapitools/openapi-generator</url>
-    <description>Equinix Fabric API Specification 4.11 generated SDK</description>
+    <description>OpenAPI Java</description>
```

Comment:
Do we need to add 'SDK' in this description?

Reply:
That's just default from the open api generator. For now we can leave as is and return to it later with a longer term fix. Just need to get this release cycle handled properly.

Reply:
Sure